darts/datasets/dataset_loaders.py

```diff
@@ -113,6 +121,29 @@ def _download_dataset(self):
                 "Could not download the dataset. Reason:" + e.__repr__()
             ) from None
 
+    def _download_zip_dataset(self):
+        os.makedirs(self._root_path, exist_ok=True)
+        extracted_dir = Path(DatasetLoader._DEFAULT_DIRECTORY, "temp_dir")
```

Comment:
It'd be safer to use [this way of creating temporary directories](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory). Also using a context manager can ensure the directory is deleted.

Reply:
I did not know about the `tempfile` library. I was using `finally` to clean up the directory, but this is much nicer.